src/core/config/types.ts

```diff
@@ -90,3 +90,12 @@ export type FaviconPack = Record<
   | 'icon-512x512.png',
   string
 >;
+
+export type AboutFeatureConfig = {
+  tabId: string;
+  assetUrl: string;
+  subTabs: {
+    tabId: string;
+    assetUrl: string;
+  }[];
+};
```

Comment:
Mmm, I would probably make these types recursive once we actually start processing these objects recursively. Otherwise it just feels like unneeded complexity